drive/drive.go

```diff
@@ -51,7 +51,8 @@ func (ud *UserDrive) PublicSpace(ctx context.Context, api fxiface.CoreAPI) (*Dri
 		Api:       api,
 		SpaceType: PublicDriveSpaceType,
 		RootCid:   ud.PublicSpaceCid,
-		RootDir:   rootDir.(files.Directory)}}, err
+		RootDir:   rootDir.(files.Directory),
+		Drive:     ud}}, err
```

Comment:
does this code compile?

Reply:
yes, why?

Reply:
I see the formatting made that line seem odd to me.